caluma/caluma_core/management/commands/migrate.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.commands import migrate
+
+from .migrate_to_prefixed_apps import Command as PrefixCommand
+
+
+class Command(migrate.Command):
+    """migrate command wrapper to add custom logic."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.pre = [self._set_app_prefix]
+
+    def _set_app_prefix(self):
+        prefix_migrator = PrefixCommand()
+        prefix_migrator.collect()
+
+        if prefix_migrator.queries:  # pragma: no cover
+            # this is already tested in test_migrate_to_prefixed_apps
+            self.stderr.write(
+                "Adding table prefix and updating django internal tables."
+            )
+            prefix_migrator.apply()
+
+    def handle(self, *args, **options):
+        for fn in self.pre:
+            fn()
```

Comment:
Why not call `_set_app_prefix()` directly?

Reply:
That's my premature optimisation. Adding complexity where none is needed ;)

I've removed it.